testsuite/integration-arquillian/tests/other/base-ui/src/test/java/org/keycloak/testsuite/ui/account2/SigningInTest.java

```diff
@@ -240,6 +240,9 @@ public void otpTest() {
         assertEquals(2, otpCredentialType.getUserCredentialsCount());
         assertUserCredential(label2, true, otp2);
 
+        assertFalse("Not set up link is visible", otpCredentialType.isNotSetUpLinkVisible());
+        assertTrue("Title is not visible", otpCredentialType.isTitleVisible());
```

Comment:
Before this, we need to disable given required action and also add an assert to check that the setup link is not visible. We could also check the user credential count didn't change after that. Something like:
```suggestion
        assertTrue("Set up link is not visible", otpCredentialType.isSetUpLinkVisible());

        RequiredActionProviderRepresentation requiredAction = new RequiredActionProviderRepresentation();
        requiredAction.setEnabled(false);
        testRealmResource().flows().updateRequiredAction(CONFIGURE_TOTP.name(), requiredAction);

        assertFalse("Set up link is visible", otpCredentialType.isSetUpLinkVisible());
        assertFalse("Not set up link is visible", otpCredentialType.isNotSetUpLinkVisible());
        assertTrue("Title is not visible", otpCredentialType.isTitleVisible());
        assertEquals(2, otpCredentialType.getUserCredentialsCount());
```